external/emulation/emulation/config.py

```diff
@@ -78,6 +87,9 @@ def _build_masks(self) -> Iterable[Mask]:
         for key, range in self.ranges.items():
             yield RangeMask(key, min=range.min, max=range.max)
 
+        for key, _slice in self.mask_emulator_levels.items():
```

Comment:
I wonder if this should be placed at the very end?  As is, it means the conservative clouds will be determined by the blended humidity state.  Probably not a big deal since the fortran levels should be "correct".

Reply:
Moved it.